optimism.tokenlist.json

```diff
@@ -179,6 +179,50 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    },
+     {
+      "chainId": 42,
+      "address": "0x24a19ee5a5c8757acdebe542a9436d9c796d1c9e",
+      "name": "0xBitcoin",
+      "symbol": "0xBTC",
+      "decimals": 8,
+      "logoURI": "https://ethereum-optimism.github.io/logos/0xBTC.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x22F24361D548e5FaAfb36d1437839f080363982B"
+      }
+    },
+    {
+      "chainId": 69,
+      "address": "0x4D772843C34E84dC8091cCe1484623445fBe6eC7",
```

Comment:
Please update to newly deployed test token with 8 decimals here 0x56B4F5f44d348EC3F07Bf1f31A3B566E5304BedE

Reply:
I am confused.  so the newly deployed test token address is 0x56B4F5f44d348EC3F07Bf1f31A3B566E5304BedE? or you want a new token deployed....